src/TConsole.cpp

```diff
@@ -288,7 +293,8 @@ TConsole::TConsole(Host* pH, ConsoleType type, QWidget* parent)
     splitter->setSizePolicy(sizePolicy);
     splitter->setOrientation(Qt::Vertical);
     splitter->setHandleWidth(3);
-    splitter->setPalette(splitterPalette);
+    auto style_sheet = QStringLiteral("background-color: rgba(0,0,0,0)");
```

Comment:
Could this and other cases be camelCase? That's the standard we follow on the C++ side.